.github/workflows/luseepy-test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     container: buddhasystem/lusee-night-pyshtools:0.1 # ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: wget "https://drive.google.com/uc?export=download&id=18hlczRVOkt2bQI-j_WzsSgXeVWGZflgA"
+      - run: wget "https://drive.google.com/uc?export=download&id=1W780SR2yMWAMO8kZZkdvAtclD_9yJnEx"
       - run: tar zxvf uc*
```

Comment:
Maybe you could use the `wget` option `-O filename` which lets you name the file? Then you don't have to use this `uc*` wildcard.

Reply:
This seems fine to merge. I gave running the beam conversion scripts a try -- it seems like the files in the Google Drive have changed since the scripts were written. The script themselves seem to run, but files like `..._Re.dat`, electric field and directivity csv are not on the drive.

All of this code seems to be missing tests, but that can come later.